lib/tasks/beautify_html.rake

```diff
@@ -0,0 +1,13 @@
+require 'htmlbeautifier'
+
+namespace :html do
+  desc "Beautify all HTML files in the project"
+  task :beautify do
+    Dir.glob("app/views/**/*.html.erb").each do |file|
+      original = File.read(file)
+      beautified = HtmlBeautifier.beautify(original)
+      File.open(file, "w") { |f| f.write(beautified) }
+      puts "Beautified: #{file}"
+    end
+  end
+end
```

Comment:
Nice stuff, I like it! 👏 

But a tip for the future, it is better to do such changes in separate PRs so it is easier to review :)

Reply:
Ok. Chat GPT did this :)